scripts/eurostat/regional_statistics_by_nuts/README.md

```diff
@@ -0,0 +1,104 @@
+# Importing EuroStat Regional Statistics by NUTS Into Data Commons
+
+> This README is a template of what an import's README might look like.
+
+Author: eftekhari-mhs
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [Import Artifacts](#import-artifacts)
+1. [Generating Artifacts](#generating-artifacts)
+
+## About the Dataset
+
+### Download URL
+
+[TSV] file is available for download from <https://ec.europa.eu/eurostat/data/database> under 'Data navigation tree > Database by themes > General and regional statistics > Regional statistics by NUTS classification (reg)'.
+
+### Overview
+> The metadata for each dataset is available online at '<https://ec.europa.eu/eurostat/cache/metadata/en/<dataset name>_esms.htm>'
+
+NUTS (Nomenclature of Territorial Units for Statistics) is a hierarchical classification for dividing regions of Europe. 
+
+Useful links: [NUTS Background](https://ec.europa.eu/eurostat/web/nuts/background), 
+[NUTS Map](https://ec.europa.eu/eurostat/web/nuts/nuts-maps)
+
+##### Imported datasets (P0): 
+
+- Population density (by NUTS3)
+- Birth, deaths, and migrations (by NUTS3)
+- Life expectancy by age and gender (by NUTS2)
+
+##### Imported datasets (P1): 
+
+- Moratality rate by cause (NUTS2)
+- Fertility rate, median and mean age of mothers at childbirth (by NUTS3)
+- Various measures of GDP (by NUTS3)
+- Education enrollment by gender (by NUTS2)
+- Education Attainment by gender (by NUTS2)
+- Employed per sector (by NUTS3)
+
+### License
+
+Eurostat has a policy of encouraging free re-use of its data, both for non-commercial and commercial purposes. 
+
+The license is available online at [here](https://ec.europa.eu/eurostat/about/policies/copyright).
+
+
+
+### Import Procedure
```

Comment:
I think I am realizing this whole section is just copied from the example? Is that right?

Reply:
yes, we should remove it.